packages/apollo/src/diff/index.ts

```diff
@@ -36,21 +40,21 @@ export interface SchemaChange {
   description: string;
 }
 
-export const format = (change: SchemaChange) => {
-  let color = (x: string): string => x;
-  if (change.type === ChangeType.FAILURE) {
-    color = chalk.red;
-  }
-  if (change.type === ChangeType.WARNING) {
-    color = chalk.yellow;
+export function format({ type, description }: ValidationResult, index: number) {
+  let color = (x: string) => x;
+  switch (type) {
+    case ValidationErrorType.FAILURE:
+      color = chalk.red;
+      break;
+    case ValidationErrorType.INVALID:
+      color = chalk.gray;
+      break;
+    case ValidationErrorType.WARNING:
+      color = chalk.yellow;
+      break;
   }
-
-  return {
-    type: color(change.type),
-    code: color(change.code),
-    description: color(change.description)
-  };
-};
+  return `    [${index}] ${color(type)}    ${description}`;
```

Comment:
What's the benefit of adding the index to the error log?

Reply:
I was already on the fence about leaving this in, so I pulled it since you're asking 😛
It really doesn't add anything useful.